packages/@o3r/test-helpers/src/utilities/package-manager.ts

```diff
@@ -187,6 +216,7 @@ export function setPackagerManagerConfig(options: PackageManagerConfig, execAppO
       // Set config to target local registry
       execFileSync('yarn', ['config', 'set', 'checksumBehavior', 'update'], execOptions);
       execFileSync('yarn', ['config', 'set', 'enableImmutableInstalls', 'false'], execOptions);
+      execFileSync('yarn', ['config', 'set', 'enableScripts', 'false'], execOptions);
```

Comment:
should we add the same for npm ? (`npm config set ignore-scripts true` I suppose)